src/roles/hunter.py

```diff
@@ -43,20 +43,20 @@ def hunter_kill(var, wrapper, message):
 @command("retract", "r", chan=False, pm=True, playing=True, phases=("night",), roles=("hunter",))
 def hunter_retract(var, wrapper, message):
     """Removes a hunter's kill selection."""
-    if wrapper.source not in KILLS and wrapper.source not in PASSED:
-        return
```

Comment:
This shouldn't be removed. It's there to prevent a hunter who hasn't killed or passed to retract.